Assets/MixedRealityToolkit/_Core/Devices/BaseController.cs

```diff
@@ -46,7 +46,7 @@ protected IMixedRealityInputSystem InputSystem
         {
             get
             {
-                if (inputSystem == null && MixedRealityManager.Instance.ActiveProfile.EnableInputSystem)
+                if (inputSystem == null && MixedRealityManager.Instance.ActiveProfile.IsInputSystemEnabled)
```

Comment:
Needs to also have a check for "MRManager.HasActiveProfile"

Reply:
If  we didn't have an active profile, how would we ever have created a controller?